### PR TITLE
fix documentation (and values) in radiationObserver.param

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request fixes wrong documentation in the `radiationObserver.param` 
The laser propagation was wrongly stated as towards `+x direction.
Thanks @Heikman for pointing me to these errors.

### CODE REVIEW

File: src/picongpu/include/simulation_defines/param/radiationObserver.param [hunk 1]
```diff
@@ -96,7 +96,7 @@ namespace picongpu
       const picongpu::float_64 phi(    my_index_phi   * delta_angle_phi   - angle_phi_start   );
 
       /* compute observation unit vector */
-      return vector_64( sinf(theta)*cosf(phi) , sinf(theta)*sinf(phi) , cosf(theta) ) ;
```
Comment (psychocoderHPC) [inline src/picongpu/include/simulation_defines/param/radiationObserver.param]: please use `mat::sin()`, `math::cos()` or maybe `math::sincos(phi)`
If you need 32bit please cast your values with `precisionCast<float_X>()` before.

**Comment by ax3l** [comment]
let me add the bug flag since its an documentation bug so users can spot it.

File: src/picongpu/include/simulation_defines/param/radiationObserver.param (hunk 1 above)
Comment (PrometheusPi) [inline src/picongpu/include/simulation_defines/param/radiationObserver.param]: @psychocoderHPC I fixed the use of `sinf` and `cosf`  - I am not sure why the now removed line does not show up as "outdated diff".

File: examples/Bunch/include/simulation_defines/param/radiationObserver.param [hunk 2]
```diff
@@ -65,7 +65,9 @@ namespace picongpu
       /* + picongpu::PI -> turn observation direction 180 degrees towards -y */
 
       /* compute observation unit vector */
-      return vector_64(sinf(theta), cosf(theta), 0.0);
+      return vector_64(math::sin(precisionCast<picongpu::float_32>(theta)),
```
Comment (psychocoderHPC) [inline examples/Bunch/include/simulation_defines/param/radiationObserver.param]: could you please use `math::sincos()`

File: examples/SingleParticleRadiationWithLaser/include/simulation_defines/param/radiationObserver.param [hunk 3]
```diff
@@ -66,7 +66,9 @@ namespace picongpu
       /* + picongpu::PI -> turn observation direction 180 degrees towards -y */
 
       /* compute observation unit vector */
-      return vector_64(sinf(theta), cosf(theta), 0.0);
+      return vector_64(math::sin(precisionCast<picongpu::float_32>(theta)),
```
Comment (psychocoderHPC) [inline examples/SingleParticleRadiationWithLaser/include/simulation_defines/param/radiationObserver.param]: could you please use `math::sincos()`

**Comment by PrometheusPi** [comment]
@psychocoderHPC I added `math::sincos` to the two simple cases as well.

File: examples/Bunch/include/simulation_defines/param/radiationObserver.param [hunk 4]
```diff
@@ -65,7 +65,10 @@ namespace picongpu
       /* + picongpu::PI -> turn observation direction 180 degrees towards -y */
 
       /* compute observation unit vector */
-      return vector_64(sinf(theta), cosf(theta), 0.0);
+      picongpu::float_32 sinTheta;
+      picongpu::float_32 cosTheta;
+      math::sincos(precisionCast<picongpu::float_32>(theta), sinTheta, cosTheta);
```
Comment (PrometheusPi) [inline examples/Bunch/include/simulation_defines/param/radiationObserver.param]: Just a **question**:
@psychocoderHPC what would happen if I avoid the `precisionCast` here. Would it run in `double` or `single` precision or would it not run at all?

Comment (psychocoderHPC) [inline examples/Bunch/include/simulation_defines/param/radiationObserver.param]: If depends on `theta`, if it is a 32bit type than float is used else double.

**Comment by psychocoderHPC** [comment]
I removed the example flag because it is more a part of the plugin radiation.

**Comment by PrometheusPi** [comment]
@psychocoderHPC  That is fine with me, I just thought @ax3l likes the `examples` flag, if he builds a new release.

**Pull request merged**